src/ydata_quality/bias_fairness/engine.py

```diff
@@ -57,7 +57,8 @@ def proxy_identification(self, th=0.5):
         if len(corrs) > 0:
             self.store_warning(
                 QualityWarning(
-                    test='Proxy Identification', category='Bias&Fairness', priority=2, data=corrs,
+                    test=QualityWarning.Test.PROXY_IDENTIFICATION,
+                    category=QualityWarning.Category.BIAS_FAIRNESS, priority=2, data=corrs,
```

Comment:
Should we also change the ints in priority for the values defined at the warnings.py priority enum?